topics/data-management/tutorials/0013_file-naming/slides.html

```diff
@@ -0,0 +1,187 @@
+---
+layout: rdmbites_slides
```

Comment:
Do you want to keep this filename @robertmand? The `0013_` bit? It'll appear in the URL, and just wanted to make sure that's what you want. Usually we would just truncate to `file-naming` because it looks nicer.

Reply:
Resolved out of band.